backend/drm/drm.c

```diff
@@ -18,6 +18,7 @@
 #include <wlr/backend/interface.h>
 #include <wlr/interfaces/wlr_output.h>
 #include <wlr/util/log.h>
+#include "util/signal.h"
```

Comment:
Well you still need to group this with the other internal includes. Group quotes separate from <>

Reply:
Double newline to separate include groups?

Reply:
No no no no no please no

Reply:
One last question: do I group stdlib/external/public/private, or external/public/private?

Reply:
<> in one group, "" in another. Use <> for stuff that is (or will be, e.g. public wlr headers) installed in /usr/include